### ISSUE TITLE
Don't refer to cookies as legacy (yet)

### ISSUE THREAD

**Issue by johnwilander** [privacycg/is-logged-in#8]
This issue is ported from https://github.com/WebKit/explainers/issues/39.

> I don't think any browser has shipped HTTP State Tokens yet, so it seems premature to assume it's the default or to call cookies legacy.

**Comment by bc-pi** [comment]
To take it a bit further and while I realize that the names of those involved may contribute to the expectation that it's real, currently HTTP State Tokens is only an expired individual draft https://tools.ietf.org/html/draft-west-http-state-tokens-00 , which doesn't carry any authority or expectation of being progressed (https://tools.ietf.org/html/draft-abr-twitter-reply-00 is a good example of this). The link to it in the IsLoggedIn explainer is to a page on mikewest.github.io, which also isn't a particularly authoritative standards development organization. Also the assumed repo for it https://github.com/mikewest/http-state-tokens, which hasn't been touched in over a year, has this disclaimer, "(Note: This isn't a proposal that's well thought out, and stamped solidly with the Google Seal of Approval. It's a collection of interesting ideas for discussion, nothing more, nothing less.)" 

So yes, it does seem a bit premature.

**Comment by samuelweiler** [comment]
"legacy" is acceptable language to describe the current state of the world when proposing changes.  Propose to close this issue.

**Comment by adamroach** [comment]
I agree with @othermaciej and @bc-pi that the _implication_ of the term "legacy" is that cookies are deprecated (or at least actively being replaced), which seems to assume facts not in evidence. I don't even see a serious effort to socialize State Tokens in the broader community yet, much less consensus to move in that direction.
